### ISSUE TITLE
Minor: Plot Components Warning

### ISSUE THREAD

**Issue by edwinnglabs** [uber/orbit#439]
**Describe the bug**
I saw a warning when I run components plot in dev branch.

**To Reproduce**
```
plot_predicted_components(predicted_df=predicted_df, date_col=date_col, 
                        plot_components=['trend', 'seasonality_7', 'seasonality_365.25'])
```

cell no. 11 under `examples/ktrlie.ipynb`

**Screenshots**
![Screen Shot 2021-05-23 at 11 54 36 AM](https://user-images.githubusercontent.com/43593379/119273067-00691880-bbbe-11eb-99d7-410f84d67c6f.png)

**Environment (please complete the following information):**
 - Python Version: 3.7
 - Matplotlib Version: 3.3.4

**Comment by Okroshiashvili** [comment]
@edwinnglabs and @wangzhishi  I'll do this fix if not yet done

**Comment by edwinnglabs** [comment]
Thank you @Okroshiashvili . Please feel free to do so.

**Comment by Okroshiashvili** [comment]
@edwinnglabs I followed the contribution guide and now I cannot push my fixes and changes. All the time I'm getting access denied error. `ERROR: Permission to uber/orbit.git denied to Okroshiashvili.
fatal: Could not read from remote repository.`

**Comment by ppstacy** [comment]
Thanks @Okroshiashvili  for helping fix this issue!

**Comment by Okroshiashvili** [comment]
You're welcome @ppstacy . I think this is not done yet. I found a bug #468 and waiting that to be fixed to continue working. Unfortunately, I cannot fix #468.
